src/node_options.cc

```diff
@@ -324,6 +324,10 @@ EnvironmentOptionsParser::EnvironmentOptionsParser() {
             "experimental WASI support",
             &EnvironmentOptions::experimental_wasi,
             kAllowedInEnvironment);
+  AddOption("[harmony_top_level_await]",
+            "",
+            &EnvironmentOptions::harmony_top_level_await);
+  Implies("--harmony-top-level-await", "[harmony_top_level_await]");
```

Comment:
If possible I would very much like this flag to be `--experimental-top-level-await`. Folks may or may not know what `harmony` implies... but generally it is referring to the V8 flag, and I'm not thrilled to be exposing it that way. It definitely makes sense that we are continuing to expose this as `--harmony-top-level-await`, as we are essentially piggy backing on the flag to make sure we wire up the implementation... but I'd prefer if we could keep the flag consistent with other module .